compose/utils.py

```diff
@@ -143,3 +144,7 @@ def parse_bytes(n):
         return sdk_parse_bytes(n)
     except DockerException:
         return None
+
+
+def generate_uid():
+    return str(uuid.uuid4().hex)
```

Comment:
This generates really long identifiers. Perhaps it would be worth making them 10 characters or so?